requirements.txt

```diff
@@ -1,8 +1,8 @@
-Django==1.9.7
+Django==1.10.2
 dj-static==0.0.6
 django-versioned-static-url==0.8
 django-static-root-finder==0.1
 django-asset-server-url==0.1
-django-template-finder-view==0.2
+django-template-finder-view==0.3
 django-unslashed==0.3.0
 django-yaml-redirects==0.4
```

Comment:
I get an error because of empty redirects yaml. Which I see you've fixed upstream - good work! Could you change this to `django-yaml-redirects==0.5.4`?